### ISSUE TITLE
pride plugin crashing if `TERM` is unset

### ISSUE THREAD

**Issue by 4e554c4c** [minitest/minitest#1009]
### Reproduction
Erasing the `TERM` environment variable causes tests to fail if the pride plugin is enabled. A reproduction in fish shell is displayed below
```fish
$ ruby -Ilib:test test/minitest/test_minitest_test.rb -p
Run options: -p --seed 54501

# Running:

.........................................................................

Fabulous run in 0.041388s, 1763.8114 runs/s, 4687.3892 assertions/s.

$ set -e TERM
$ ruby -Ilib:test test/minitest/test_minitest_test.rb -p
/home/pounce/programming/minitest/lib/minitest/pride_plugin.rb:13:in `plugin_pride_init': undefined method `match?' for nil:NilClass (NoMethodError)
	from /home/pounce/programming/minitest/lib/minitest.rb:131:in `block in init_plugins'
	from /home/pounce/programming/minitest/lib/minitest.rb:125:in `each'
	from /home/pounce/programming/minitest/lib/minitest.rb:125:in `init_plugins'
	from /home/pounce/programming/minitest/lib/minitest.rb:281:in `run'
	from /home/pounce/programming/minitest/lib/minitest.rb:85:in `block in autorun'
```

**Comment by zenspider** [comment]
Okay... what the hell... did GHA remove TERM?

**Comment by 4e554c4c** [comment]
> Okay... what the hell... did GHA remove TERM?

yeah seems like it. i don't remember this being a problem before

**Comment by 4e554c4c** [comment]
fixed in 9e7c58d
